game_controller.py

```diff
@@ -1,7 +1,81 @@
+import random
+
+import user_interaction as user_inter
+from citizens.citizen import Citizen
+from citizens.player import Player
+from citizens.spy import Spy
+from game import Game
+
+
 class GameController:
     def __init__(self) -> None:
+
+        self.citizens_dict = {
+            "Охотник": "Ловушка",
+            "Врач": "Антидот",
+            "Сутенер": "Клофелинщица"
+        }
+
+        self.first_user = 'GvinP'
+        self.second_user = 'Runmaget'
+        self.available_citizens = []
```

Comment:
What kind of possibilities it might open?

Reply:
> Type hints.
> It is not obvious List of which objects acailable_citizens is

Done.

Reply:
> SHOULD BE DISCUSSED.
> 
> Also, available_citizens used only during initialization process and after this it is useless. This is confusing.
> This field should be removed or renamed.
> 
> I think - the best approach - to pass local variable - available_citizens as an argument to create_player and create_spy methods.
> This will also open new possibilities for testing

 Changed this field to the local field, which passed as an argument to  create_citizens, create_players, and create_spy methods.

Reply:
> Type hints.
> It is not obvious List of which objects acailable_citizens is

Done

Reply:
> SHOULD BE DISCUSSED.
> 
> Also, available_citizens used only during initialization process and after this it is useless. This is confusing.
> This field should be removed or renamed.
> 
> I think - the best approach - to pass local variable - available_citizens as an argument to create_player and create_spy methods.
> This will also open new possibilities for testing

available_citizens was added as a local variable, but it need to be discussed.